lib/Dialect/CGGI/IR/CGGIOps.td

```diff
@@ -99,4 +97,23 @@ def CGGI_Lut3Op : CGGI_LutOp<"lut3", [AllTypesMatch<["a", "b", "c", "output"]>]>
   let results = (outs LWECiphertextLike:$output);
 }
 
+def CGGI_LutLinCombOp : CGGI_LutOp<"lut_lincomb", [SameOperandsAndResultType]> {
+  let summary = "A variadic-input lookup table with inputs prepared via linear combination.";
+  let arguments = (ins
+    Variadic<LWECiphertext>:$inputs,
+    DenseI32ArrayAttr:$coefficients,
+    Builtin_IntegerAttr:$lookup_table
+  );
+  let description = [{
+    An op representing a lookup table applied to an arbitrary number of input
+    ciphertexts, which are combined according to a static linear combination
+    attached to the op.
+
+    E.g., where the semantics of `cggi.lut3` combine the inputs into a single
+    ciphertext using products with plaintexts and sums, this op assumes that
+    work has been done in advance, using `lwe.add` and `lwe.mul_scalar`.
```

Comment:
Is this still true? It seems like the add and mul_scalars are represented in the `coefficients` attribute.

It may also be worth mentioning that the user is expected to ensure that the linear combination does not overflow the error bits.

Reply:
Fixed